client/fidelity/compliance-review/views/compliance-review.html

```diff
@@ -10,12 +10,26 @@
                 </span>
             </span>
             <span class="subnav__stretch-bar subnav__stretch-bar--right">
+                <div class="subnav__button-stack subnav__button-stack--padded">
+                <button
+                    ng-class="{ btn: true, 'btn--primary': search.showEnglishOnly }"
+                    ng-click="search.showEnglishOnly = true">
+                    {{ 'English only' | translate }}
+                </button>
+                <button
+                    ng-class="{ btn: true, 'btn--primary': !search.showEnglishOnly }"
+                    ng-click="search.showEnglishOnly = false">
+                    {{ 'All languages' | translate }}
```

Comment:
Never use `| translate` filter for static values, it doesn't extract the phrase. Instead add a `<span translate>All languages</span>`